packages/core/src/navigation-item/NavigationItem.tsx

```diff
@@ -75,34 +130,69 @@ export const NavigationItem = forwardRef<HTMLDivElement, NavigationItemProps>(
       "--saltNavigationItem-level": `${level}`,
     };
 
+    const isParent = parent || href === undefined;
+    const elementProps = {
+      className: clsx(
+        withBaseName("wrapper"),
+        {
+          [withBaseName("active")]: active || blurActive,
+          [withBaseName("blurActive")]: blurActive,
+          [withBaseName("rootItem")]: level === 0,
+        },
+        withBaseName(orientation)
+      ),
+      children: (
+        <>
+          <span className={withBaseName("label")}>{children}</span>
+          {isParent ? (
+            <ExpansionIcon expanded={expanded} orientation={orientation} />
+          ) : null}
+        </>
+      ),
+    };
+    let parentProps: Partial<ComponentPropsWithoutRef<"button">> = {};
+    let linkProps: Partial<ComponentPropsWithoutRef<"a">> = {};
+    if (isParent) {
+      const handleExpand = onExpand
+        ? (event: MouseEvent<HTMLButtonElement>) => {
+            event.stopPropagation();
+            onExpand(event);
+          }
+        : undefined;
+      parentProps = {
+        ...elementProps,
+        "aria-label": "expand",
+        "aria-expanded": expanded,
+        onClick: handleExpand,
+      };
+    } else {
+      linkProps = {
+        ...elementProps,
+        href,
+        "aria-label": "change page",
```

Comment:
```suggestion
```
Not sure why an aria-label was added?

Reply:
![image](https://github.com/jpmorganchase/salt-ds/assets/12938082/69a1988c-06e1-4500-8d73-511876840eda)
This breaks accessibility